critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
I am trying do that and also looking for some other plugin which might be more suitable.